test/system/030-run.bats

```diff
@@ -1270,7 +1274,8 @@ EOF
     myvolume=my-volume-$(safename)
     run_podman volume create $myvolume
     mkdir $romount/volume
-    run_podman run --rm --uidmap=0:1000:10000 -v volume:/volume:idmap --rootfs $romount stat -c %u:%g /volume
+    chown 1000:1000 $romount/volume
+    run_podman run --rm --uidmap=0:1000:10000 -v $myvolume:/volume:idmap --rootfs $romount stat -c %u:%g /volume
```

Comment:
this new version works well for me, but as you I am on F40.

Not sure why it fails on Debian, I've checked the kernel and it is recent enough that support for idmapped mounts on tmpfs is there (and it would fail with EINVAL anyway).  If it still fails, I may need to take a look on Debian

Reply:
It fails on absolutely everything, not just debian. Most recent push includes `cat /etc/sub?id` for debugging but it'll take 30m for it to fail

Reply:
hm.. if I run the test manually on a 'debian-13 root host sqlite' VM using the code in this PR, the test passes:

```
# uname -a
Linux gscrivano-debian-c20240708t152000z-f40f39d13 6.9.8-cloud-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.9.8-1 (2024-07-07) x86_64 GNU/Linux

root@gscrivano-debian-c20240708t152000z-f40f39d13:/var/tmp/go/src/github.com/containers/podman# bats -f 'podman run - rootfs with idmapped mounts' test/system/030-run.bats
1..1
ok 1 [030] podman run - rootfs with idmapped mounts
```

Is there anything done differently that I am missing?

Reply:
I don't know how to reproduce the failure. It all passes on my laptop and on 1minutetip.

One recommendation: `hack/bats` is infinitely better than running `bats` directly. It does a lot of important setup.
```console
# hack/bats --root 030:idmapped
.... runs all tests that match the string "idmapped" in the 030 file

# hack/bats --help
```

Reply:
Keep in mind we have a tmpfs path mounted on the container storage now

Reply:
Good hypothesis, thanks. Unfortunately, test passes (1mt) even with tmpfs `/var/lib/containers`. Test also passes if I run all `030` tests, or even just run `hack/bats`, suggesting that it's not something caused by a previous test.